README.md

```diff
@@ -1,35 +1,458 @@
-Ballerina JDBC Library
-===================
+# Ballerina JDBC Library
+
+[![Build](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/build-timestamped-master.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/actions/workflows/trivy-scan.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-java.jdbc.svg)](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/commits/master)
+[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/java.jdbc.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fjava.jdbc)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-java.jdbc/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-java.jdbc)
+
+This package provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Comment:
```suggestion
This library provides the functionality that is required to access and manipulate data stored in any type of relational database,
```

Reply:
Fixed in [e697738](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/347/commits/e697738db73e0796ce31139cbc8ac128975d0716)